galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.ceci.manager.ivt/src/main/java/dev/galasa/cicsts/ceci/manager/ivt/CECIManagerIVT.java

```diff
@@ -75,6 +80,22 @@ public void testNotNull() throws CicstsManagerException  {
       assertThat(batch).isNotNull();
    } 
    
+   /**
+    * Starts the CECI session.
+    * 
+    * @throws CicstsManagerException 
+    * @throws CeciException 
+    */
+   @Test
+   public void testStartCeciSession() throws CeciException, CicstsManagerException {
+	   logger.info("Testing starting a CECI session");
```

Comment:
you've removed similar lines to this in other test methods!

Reply:
I am going to remove all of the logger messages that don't really add anything more than what the java doc comment and the method name do :)